aip/general/0135.md

```diff
@@ -179,6 +161,45 @@ message DeleteBookRequest {
 If the etag is provided and does not match the server-computed etag, the
 request **must** fail with a `FAILED_PRECONDITION` error code.
 
+**Note:** Declarative-friendly resources (AIP-128) **must** provide the `etag`
+field for Delete requests.
+
+### Delete if existing
+
+[^reviewing]
+
+If the service uses client-assigned resource names, `Delete` methods **may**
+expose a `bool allow_missing` field, which will cause the method to succeed in
+the event that the user attempts to delete a resource that is not present (and
+no-op):
```

Comment:
Here and a bit later you use no-op as a verb... I'm not familiar with that usage, and although I can readily understand it, I wonder whether it might be harder for those with English as a non-primary language.

Something like "in which case the request is a no-op"?

Reply:
Done.